### PR TITLE
Fix/wp seo test

### PR DESCRIPTION
**Pull request by stratease**
Splitting WP SEO test suite into it's own workflow, so we can change the required version of WP core.

### CODE REVIEW

**Review by bordoni — DISMISSED** [review]
I like this change, I feel like these integrations should all be on their own.

**Comment by bordoni** [comment]
Change a PHP File to trigger the tests.

File: .github/workflows/tests-php-seo-plugin.yml [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+on:
+  pull_request:
+    paths:
+      - 'tests/**.php'
+      - 'src/**.php'
+      - '*.php'
+      - 'composer.json'
+      - 'composer.lock'
+      - 'codeception.dist.yml'
+      - '.github/workflows/tests-php.yaml'
```
Comment (bordoni) [inline .github/workflows/tests-php-seo-plugin.yml:11]:
```suggestion
      - '.github/workflows/tests-php-seo-plugin.yml'
```

**Comment by stratease** [comment]
@bordoni If you can approve and merge - [the failed test](https://github.com/the-events-calendar/the-events-calendar/actions/runs/7729207360/job/21074048040?pr=4469) is fixed in https://github.com/the-events-calendar/the-events-calendar/pull/4467

**Review by lucatume — APPROVED** [review]

**Pull request merged**